### PR TITLE
Display access packages search

### PR DESCRIPTION
**Pull request by allinox**
## Description
This PR adds a package delegation modal to the access package section, displaying search for access packages.
Specifically, it:
- Adds new BFF Controller, Server, and Client for Access Packages, with an endpoint for access package search.
  - The search endpoint currently does not support filtering as details around filtering is somewhat uncertain atm.
- Adds mock for the AccessPackageClient and the search endpoint
- Adds BFF tests for new controller and enpoint. (All input is valid for the new endpoint and the search operation itself is up backend's task so there are only two new tests, both of which are positive.)
- Splits the delegation modal into two "modes", one for each type of delegation. I wasn't completely sure about this or if it would be better to just create completely separate modals, but there is a lot of overlap in form and functionality so I figured I'd give it a try. Also, if we are to (in the future) combine the delegation processes into a single modal with search across delegation types, keeping the same wrapping will support that.
- Display access package data gained from search. Minimal effort is spent on styling here though, as they are to be represented using the ListItem component from the component library, which is to be added soon.
- Added new handler for frontend data mocking and display in storybook

image

## Related Issue(s)
- [79](https://github.com/Altinn/altinn-authorization-tmp/issues/79)

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- [x] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/AccessArea.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        /// </summary>
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Name
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Descriotion
```
Comment (sonwit) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/AccessArea.cs:19]:
```suggestion
        /// Description
```

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/AccessPackage.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        /// </summary>
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Name
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Descrioption
```
Comment (sonwit) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/AccessPackage.cs:19]:
```suggestion
        /// Description
```

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/Frontend/AccessAreaFE.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        /// </summary>
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Name
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Descriotion
```
Comment (sonwit) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/Frontend/AccessAreaFE.cs:19]:
```suggestion
        /// Description
```

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/AccessArea.cs (hunk 1 above)
Comment (allinox) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/AccessArea.cs:19]: 🤦‍♀️

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/AccessPackage.cs (hunk 2 above)
Comment (allinox) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/AccessPackage.cs:19]: 🤦‍♀️

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/Frontend/AccessAreaFE.cs (hunk 3 above)
Comment (allinox) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/Frontend/AccessAreaFE.cs:19]: Herregud, hva sier det om meg som klarer å skrive det samme ordet feil tre ganger på forskjellig vis 🙈

Comment (sonwit) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/AccessPackage/Frontend/AccessAreaFE.cs]: Haha! 😂 Neida, det sier bare at du fokuserer på de viktige tingene, som å lage fantastisk kode! 😜  Hvem bryr seg om litt skriveleifer i dokumentasjonen😎!

**Pull request merged**